packages/js/components/src/phone-number-input/utils.ts

```diff
@@ -0,0 +1,133 @@
+/**
+ * External dependencies
+ */
+import { keyBy, mapValues, sortBy } from 'lodash';
```

Comment:
Let's avoid use of lodash. It carries significant weight.

Reply:
Sure! I also prefer to avoid it for weight reasons, but as we load it from the WP env, I used it for convenience 🙂 Got rid of it in b6b3546